mkdocs.yml

```diff
@@ -145,6 +163,9 @@ extra:
   version:
     method: mike
     provider: mike
+  var:
+    prev_git_tag: "v0.0.0"
+    chart_version: 0.0.1
```

Comment:
This probably should be Trivy chart version , or I assume this section can be removed if it wasn't exist before